Lib/test/test_generated_cases.py

```diff
@@ -1371,6 +1371,24 @@ def test_stack_save_only(self):
         with self.assertRaises(SyntaxError):
             self.run_cases_test(input, output)
 
+    def test_instruction_size_macro(self):
+        input = """
+        inst(OP, (--)) {
+            frame->return_offset = INSTRUCTION_SIZE;
+        }
+        """
+
+        output = """
+        TARGET(OP) {
+            frame->instr_ptr = next_instr;
+            next_instr += 1;
+            INSTRUCTION_STATS(OP);
+            frame->return_offset = 1 ;
+            DISPATCH();
+        }
+        """
+        self.run_cases_test(input, output)
+
```

Comment:
Add a test for another instruction that is a different length, but uses `OP`:
```
macro(OP2) = unused/1 + OP;
```

Reply:
Added!

Reply:
That test should fail, as the `INSTRUCTION_SIZE` is inconsistent.
The tier 2 `INSTRUCTION_SIZE` cannot be both 1 and 2.

Reply:
Right, sorry I mistakenly thought it should only fail for tier 2 hence I was only checking the instruction in the tier 2 generator. I extended the check to tier 1 as well and the test now fails as expected.